resources/references/core-reference/administration-reference/utils.md

```diff
@@ -0,0 +1,109 @@
+# Utils reference
+
+This is an overview of all the utility functions bound to the shopware global object.
+Utility functions provide many useful shortcuts for common tasks, see how to use them in your plugin [here](./../../../../guides/plugins/plugins/administration/using-utils.md).
+
+## General functions
+
+| Function | Description | Link
+| :--- | :--- | :---- |
+| createId | Returns a uuid string in hex format. Generated with [uuid](https://www.npmjs.com/package/uuid)
+| throttle | Creates a `throttled` function that only invokes `func` at most once per every `wait` milliseconds. | [link](https://lodash.com/docs/4.17.15#throttle)
+| debounce | Creates a `debounced` function that delays invoking `func` until after `wait` milliseconds have elapsed since the last time the `debounced` function was invoked. | [link](https://lodash.com/docs/4.17.15#debounce)
+| flow     | Creates a function that returns the result of invoking the given functions with the `this` binding of the created function, where each successive invocation is supplied the return value of the previous. | [link](https://lodash.com/docs/4.17.15#flow)
+| get      | Gets the value at `path` of `object` | [link](https://lodash.com/docs/4.17.15#get) |
+
+## Object
+
+| Function | Description | Link
+| :--- | :--- | :---- |
+| deepCopyObject  | Deep copy an object |
```

Comment:
Shall we use `code` syntax for all functions? 🤔

Reply:
nah